src/client/app_configuration_client.rs

```diff
@@ -12,316 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::client::cache::ConfigurationSnapshot;
+use crate::Result;
+
+use crate::client::feature_proxy::FeatureProxy;
 use crate::client::feature_snapshot::FeatureSnapshot;
-pub use crate::client::feature_proxy::FeatureProxy;
-use crate::client::http;
+use crate::client::property_proxy::PropertyProxy;
 use crate::client::property_snapshot::PropertySnapshot;
-pub use crate::client::property_proxy::PropertyProxy;
-use crate::errors::{ConfigurationAccessError, Error, Result};
-use crate::models::Segment;
-use std::collections::{HashMap, HashSet};
-use std::net::TcpStream;
-use std::sync::{Arc, Mutex};
-use std::thread;
-use std::time::Duration;
-use tungstenite::stream::MaybeTlsStream;
-use tungstenite::Message;
-use tungstenite::WebSocket;
-
-/// App Configuration client for browsing, and evaluating features and
-/// properties.
-#[derive(Debug)]
-pub struct AppConfigurationClient {
-    pub(crate) latest_config_snapshot: Arc<Mutex<ConfigurationSnapshot>>,
-    pub(crate) _thread_terminator: std::sync::mpsc::Sender<()>,
-}
```

Comment:
All of this, and the implementation, has been moved to `src/client/app_configuration_ibm_cloud.rs`